packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```diff
@@ -180,11 +217,21 @@ popd
 
 %files
 %{_cross_attribution_file}
+%license fabricmanager-linux-%{fm_arch}-%{tesla_ver}-archive/third-party-notices.txt
 %dir %{_cross_libexecdir}/nvidia
 %dir %{_cross_libdir}/nvidia
 %dir %{_cross_datadir}/nvidia
 %dir %{_cross_libdir}/modules-load.d
 %dir %{_cross_factorydir}%{_cross_sysconfdir}/drivers
+%dir %{_cross_factorydir}%{_cross_sysconfdir}/nvidia
+%{_cross_bindir}/nvidia-fabricmanager
+%{_cross_bindir}/nvswitch-audit
+%{_cross_datadir}/nvidia/nvswitch/dgxa100_hgxa100_topology
+%{_cross_datadir}/nvidia/nvswitch/dgx2_hgx2_topology
+%{_cross_datadir}/nvidia/nvswitch/dgxh100_hgxh100_topology
+%{_cross_datadir}/nvidia/nvswitch/dgxh800_hgxh800_topology
+%{_cross_factorydir}%{_cross_sysconfdir}/nvidia/fabricmanager.cfg
+%{_cross_unitdir}/nvidia-fabricmanager.service
```

Comment:
One awkward bit here is that the Fabric Manager binaries are actually specific to the Tesla subpackage, and here they're in the main package.

The other awkward piece is that if we ever did have multiple Tesla driver subpackages, we'd probably want to use a common config file and unit.

So it might be better to adjust as follows:
```
%files tesla-%{tesla-major}
%license %{license_file} fabricmanager-linux-%{fm_arch}-%{tesla_ver}-archive/third-party-notices.txt

# Binaries
...
%{_cross_libexecdir}/nvidia/tesla/bin/nvidia-fabricmanager
%{_cross_libexecdir}/nvidia/tesla/bin/nvswitch-audit

...
# nvswitch topologies
%{_cross_datadir}/nvidia/tesla/nvswitch/dgxa100_hgxa100_topology
...
```

And add a new package that the tesla subpackage depends on with the systemd unit and config:
```
%files -n fabric-manager
%{_cross_factorydir}%{_cross_sysconfdir}/nvidia/fabricmanager.cfg
%{_cross_unitdir}/nvidia-fabricmanager.service
```

Since the binary paths and nvswitch topology directory would change, you'd need a couple edits to the service unit and the config.

Currently the main package includes all of `%{_cross_libdir}/systemd/system/` but you would need to change that to a list of explicit includes, or you would end up including `nvidia-fabricmanager.service` there also.

Reply:
changed as suggested.